.github/workflows/release.yml

```diff
@@ -74,3 +74,25 @@ jobs:
           body: "<insert release notes here>"
           token: ${{ secrets.GITHUB_TOKEN }}
           draft: true
+
+  release-version:
+    name: Upload version.json
+    runs-on: ubuntu-20.04
+    needs:
+     - setup
+     - release
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f
+        with:
+          ref: ${{ needs.setup.outputs.branch-name }}
+
+      - name: Upload version.json to S3 bucket
```

Comment:
I think this logic should work. Keep in mind that the runners come pre-installed with `awscli`. You can see a similar implementation [here](https://github.com/bitwarden/clients/blob/master/.github/workflows/release-desktop.yml#L98).

Reply:
Thanks! Using AWS CLI will be cleaner and also faster. Changed ;)

Reply:
Thanks! Using AWS CLI will be cleaner and also faster. Changed ;)